tidb-cloud/release-notes-2022.md

```diff
@@ -8,6 +8,24 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2022.
 
+## October 18, 2022
+
+**General changes**
+
+* For Dedicated Tier clusters, the minimum node storage of TiKV and TiFlash is changed from 500 GiB to 200 GiB. This will be more cost-effective for users whose workload is in small data volumes.
```

Comment:
```suggestion
* For Dedicated Tier clusters, the minimum storage size of a TiKV or TiFlash node is changed from 500 GiB to 200 GiB. This will be more cost-effective for users whose workload is in small data volumes.
```